### DISCUSSION TITLE
red arrow in tooltip

### DISCUSSION THREAD

**Discussion by vetalbayrak** [Akryum/v-tooltip#669] [Q&A]
![Adsız](https://user-images.githubusercontent.com/63376084/78839666-99ac8e00-7a01-11ea-84fe-5de782397747.png)

**Comment by jhonnattan123** [comment]
the color of that arrow depends on this css property, surely you have it modified
```
.tooltip.popover .popover-arrow {
    border-color: rgba(0, 0, 0, 0.2);
}
```
